styles.js

```diff
@@ -5,16 +5,7 @@ export default createGlobalStyle`
   *::before,
   *::after {
     box-sizing: border-box;
-  font-family: monda;
-  }
-  body {
-    margin: 0;
-  }
-  main {
-    position: relative;
-    top: 5rem;
-    margin: 0 auto;
-    max-width: 368px;
-    padding: 0.5em 1em;
+    color: darkblue;
+    font-family: arial;
```

Comment:
You should exclusively use styled-components. There is a function called createGlobalStyle with which you can do that. There should be a part about this in the styled-components handout.